experimenter/experimenter/experiments/migrations/0275_nimbusexperiment__excluded_live_deliveries_and_more.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 5.1.1 on 2024-11-29 06:44
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("experiments", "0274_nimbusbranch_firefox_labs_title_and_more"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="nimbusexperiment",
+            name="_excluded_live_deliveries",
+            field=models.JSONField(
+                blank=True,
+                default=list,
+                null=True,
+                verbose_name="Excluded Live Deliveries",
+            ),
+        ),
+        migrations.AddField(
+            model_name="nimbusexperiment",
+            name="_feature_has_live_multifeature_experiments",
+            field=models.JSONField(
+                blank=True,
+                default=list,
+                null=True,
+                verbose_name="Feature Has Live Multifeature Experiments",
+            ),
+        ),
+        migrations.AddField(
+            model_name="nimbusexperiment",
+            name="_live_experiments_in_namespace",
+            field=models.JSONField(
+                blank=True,
+                default=list,
+                null=True,
+                verbose_name="Live Experiments In Namespace",
+            ),
+        ),
+    ]
```

Comment:
I think we can avoid adding the field and can just reuse the methods we have for that and then use it as a property